src/vs/workbench/contrib/files/browser/views/explorerViewer.ts

```diff
@@ -748,17 +748,39 @@ function getFileOverwriteConfirm(name: string): IConfirmation {
 	};
 }
 
-function getMultipleFilesOverwriteConfirm(files: URI[]): IConfirmation {
-	if (files.length > 1) {
-		return {
-			message: localize('confirmManyOverwrites', "The following {0} files and/or folders already exist in the destination folder. Do you want to replace them?", files.length),
-			detail: getFileNamesMessage(files) + '\n' + localize('irreversible', "This action is irreversible!"),
-			primaryButton: localize({ key: 'replaceButtonLabel', comment: ['&& denotes a mnemonic'] }, "&&Replace"),
-			type: 'warning'
-		};
-	}
+function getFileOverwriteOrSkip(file: URI): { severity: Severity, message: string, buttons: string[], options: IDialogOptions } {
+	const confirmDialog = getFileOverwriteConfirm(basename(file));
+
+	return {
+		severity: Severity.Warning,
+		message: confirmDialog.message,
+		buttons: [
+			confirmDialog.primaryButton!,
+			localize({ key: 'skipButtonLabel', comment: ['&& denotes a mnemonic'] }, "&&Skip"),
+			localize({ key: 'cancelButtonLabel', comment: ['&& denotes a mnemonic'] }, "&&Cancel"),
+		],
+		options: {
+			cancelId: 2,
+			detail: confirmDialog.detail
+		}
+	};
+}
 
-	return getFileOverwriteConfirm(basename(files[0]));
+function getMultipleFilesOverwriteOrSkip(files: URI[]): { severity: Severity, message: string, buttons: string[], options: IDialogOptions } {
+	return {
+		severity: Severity.Warning,
+		message: localize('confirmManyOverwrites', "The following {0} files and/or folders already exist in the destination folder?", files.length),
+		buttons: [
+			localize({ key: 'replaceAllButtonLabel', comment: ['&& denotes a mnemonic'] }, "&&Replace all"),
+			localize({ key: 'skipAllButtonLabel', comment: ['&& denotes a mnemonic'] }, "&&Skip all"),
+			localize({ key: 'decideAllButtonLabel', comment: ['&& denotes a mnemonic'] }, "&&Decide invidivually"),
```

Comment:
Let's skip this option "Decide individually". Makes the whole thing overly comples and I do not like the idea of showing a lot of dialogs. Windows explorer does this by showin a dialog with a bunch of checkmarks - which we do not want to do right now.